app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -439,4 +439,21 @@
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_start">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_end">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_widget_switch_compat_margin_end">4dp</dimen>
+
+  <!-- AdministratorControlAccountActionsView  -->
+  <dimen name="administrator_control_account_actions_view_account_actions_text_view_padding_start">16dp</dimen>
+  <dimen name="administrator_control_account_actions_view_account_actions_text_view_padding_end">16dp</dimen>
+  <dimen name="administrator_control_account_actions_view_log_out_text_view_padding_start">16dp</dimen>
+  <dimen name="administrator_control_account_actions_view_log_out_text_view_padding_end">16dp</dimen>
+
+  <!-- AdministratorControlsFragment -->
+  <dimen name="administrator_controls_fragment_padding_bottom">80dp</dimen>
+
+  <!-- AdministratorControlsGeneralView -->
+  <dimen name="administrator_controls_general_view_general_text_view_padding_start">16dp</dimen>
+  <dimen name="administrator_controls_general_view_general_text_view_padding_top">20dp</dimen>
+  <dimen name="administrator_controls_general_view_general_text_view_padding_end">16dp</dimen>
+  <dimen name="administrator_controls_general_view_edit_account_text_view_padding_start">16dp</dimen>
+  <dimen name="administrator_controls_general_view_edit_account_text_view_padding_end">16dp</dimen>
```

Comment:
These dimens can be removed for `AdministratorControlsGeneralView` from this file as they match with that of `values/dimens.xml` file.

Reply:
done